### PR TITLE
You can now play with the syndicate balloon

### PR DESCRIPTION
**Pull request by Citinited**
Adds some messages for playing with the syndicate balloon. Some suggestions for more messages / handling how others see the messages (currently it only displays one message) would be nice.

:cl:
tweak: Using a syndicate balloon while it's in your hand allows you to play with it.
/:cl:

### CODE REVIEW

**Comment by davidchan13** [comment]
Does this come with a 1% (or greater) chance of releasing the balloon and watching it slowly drift up towards the ceiling (effectively becoming deleted?)

**Comment by SamHPurp** [comment]
How fast can this be spammed?

**Comment by Citinited** [comment]
@davidchan13 No 

@SamHPurp As fast as you can click, though I'm not sure that's a problem given all the messages just condense down into one line.

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -114,6 +114,14 @@
 	icon_state = "syndballoon"
 	item_state = "syndballoon"
 	w_class = WEIGHT_CLASS_BULKY
+	var/lastused = null
+
+/obj/item/toy/syndicateballoon/attack_self(mob/user)
+	if(world.time - lastused < CLICK_CD_MELEE)
+		return
+	var/playverb = pick("bat [src]", "tug on [src]'s string", "play with [src]")
+	user.visible_message("<span class = 'notice'>[user] plays with [src].</span>", "<span class = 'notice'>You [playverb].</span>")
```
Comment (tigercat2000) [inline code/game/objects/items/toys.dm]: No spaces between `class = '`, please make it `class='`

Comment (Citinited) [reply code/game/objects/items/toys.dm]: 👍

**Comment by SamHPurp** [comment]
Thanks @Citinited

**Comment by KingPhilipIII** [comment]
Now the balloon is good for more than getting tased or beaten to death by vigilantes. 👌

I remember one round I bought the balloon after I murdered my targets with my bare hands and a chemist saw me and just started attacking me and when security came and arrested him he just kept shouting “He’s a traitor, just trust me!”

**Pull request merged**